website/assets/js/pages/handbook/basic-handbook.page.js

```diff
@@ -46,6 +47,31 @@ parasails.registerPage('basic-handbook', {
       }
     }
 
+    // If this is the handbook landing page, we'll generate the page links using the `linksForHandbookIndex` array that each handbook page has
+    if(this.isHandbookLandingPage) {
+      let handbookPages = [];
+      for (let page of this.markdownPages) {
+        if(_.startsWith(page.url, '/handbook') && page.title !== 'Readme.md') {
```

Comment:
Does this capitalization work?  I think it's README.md.

I would test to make sure this is working as expected though, since if it looked like this was working, then maybe the conditional needs to be re-checked.

Reply:
@mikermcneil It does, top-level readme files use a fallback title of "Readme.md" https://github.com/fleetdm/fleet/blob/44d9c9b9046ffa7fba12ee9a029a538bd74e5f67/website/scripts/build-static-content.js#L159-L167

Reply:
@eashaw With that capitalization?  I don't see that in the linked code.  I could be missing something though.


That said, I would consider changing this to:
```js
!page.title.match(/^readme\.md$/i)
```

That way, it won't matter what the capitalization is.

Reply:
@mikermcneil the capitalization is coming from the `sails.helpers.strings.toSentenceCase()`.

Changing it to a case-insensitive match makes sense, I've updated this PR with that change.